frontend/src/views/WheresMyGene/common/store/reducer.ts

```diff
@@ -181,7 +182,8 @@ function selectOrganism(
     ...state,
     selectedGenes: [],
     selectedOrganismId: action.payload,
-    selectedTissues: [],
+    selectedTissues: EMPTY_ARRAY,
+    selectedFilters: EMPTY_FILTERS,
```

Comment:
we need to reset filters when selecting a new organism too. This way we don't need to manually dispatch an event to reset the state in `Heatmap`